<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-D5KJBUL" ignored="0" tests="2" failures="1" timestamp="02 Jul 2018 06:27:35 GMT" time="145.265" errors="0">
  <testcase name="accountLogin" time="41.725" classname="BulkInviteValidationtwo"/>
  <testcase name="bulkInvite_URL_FileTypeValidation" time="84.09" classname="BulkInviteValidationtwo">
    <failure type="java.io.IOException" message="Cannot run program &amp;quot;C:\Autoit\bulkupload2008.exe&amp;quot;: CreateProcess error=2, The system cannot find the file specified">
      <![CDATA[java.io.IOException: Cannot run program "C:\Autoit\bulkupload2008.exe": CreateProcess error=2, The system cannot find the file specified
at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
at java.lang.Runtime.exec(Runtime.java:620)
at java.lang.Runtime.exec(Runtime.java:450)
at java.lang.Runtime.exec(Runtime.java:347)
at BulkInviteValidationtwo.bulkInvite_URL_FileTypeValidation(BulkInviteValidationtwo.java:66)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
at java.lang.ProcessImpl.create(Native Method)
at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
at java.lang.ProcessImpl.start(ProcessImpl.java:137)
at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
... 29 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- bulkInvite_URL_FileTypeValidation -->
</testsuite>
